x/cronos/keeper/keeper.go

```diff
@@ -24,12 +38,30 @@ func NewKeeper(
 	cdc codec.Codec,
 	storeKey,
 	memKey sdk.StoreKey,
+	paramSpace paramtypes.Subspace,
+	evmSpace paramtypes.Subspace,
+	bankKeeper types.BankKeeper,
+	transferKeeper types.TransferKeeper,
 	// this line is used by starport scaffolding # ibc/keeper/parameter
 ) *Keeper {
+
+	// set KeyTable if it has not already been set
+	if !paramSpace.HasKeyTable() {
+		paramSpace = paramSpace.WithKeyTable(types.ParamKeyTable())
+	}
+
+	if !evmSpace.HasKeyTable() {
+		paramSpace = paramSpace.WithKeyTable(evmTypes.ParamKeyTable())
```

Comment:
- `evmSpace` and `paramSpace` don't match?
- Should we initialize EVM param space here? maybe better to get evm params through `EvmKeeper`?

Reply:
yeah for now the Keeper does not need `EvmKeeper` that is why I am only passing the `evmSpace`. You can make this change in your PR if you are planning to add the EvmKeeper to this keeper